saleor/graphql/order/mutations/orders.py

```diff
@@ -314,10 +335,10 @@ def perform_mutation(cls, _root, info, **data):
             ),
         )
 
-        clean_order_update_shipping(order, method)
+        shipping_price = info.context.plugins.calculate_order_shipping(order)
```

Comment:
We cannot calculate the shipping price and assign it to the `order.shipping_price` field before assigning `shipping_method`, because some plugins check if `shipping_method` is assigned to order (e.g. vatlayer).

So if the shipping method is assigned to order for the first time, we will get the wrong `shipping_price` value.

Reply:
@IKarbowiak 
Ok, I've switched it back to how it was. Also added a test.